ulog/builder.go

```diff
@@ -87,6 +96,7 @@ func (lb *LogBuilder) WithSentryHook(hook *sentry.Hook) *LogBuilder {
 }
 
 // Build the ulog logger for use
+// TODO: build should return `(Log, error)` in case we can't properly instantiate
```

Comment:
Why a TODO and not part of this PR? What are the cases where we can't properly instantiate?